woocommerce/woocommerce-android
# Issue/3176 shipping label refund changes

Fixes #3176 by modifying designs if shipping labels are refunded.

**This PR is in draft till [this PR](https://github.com/woocommerce/woocommerce-android/pull/3283) can be reviewed and merged.**

#### Changes
- The Products card should be displayed with products that are NOT associated with any shipping labels.
- The product section in the Shipping Labels card should be hidden if the shipping label is refunded.
- Changed the order of the products and refunds card in the Order detail screen. Previously, it was displayed below the shipping labels section but it should actually be displayed above the shipping labels section.

#### Screenshots
<img src="https://user-images.githubusercontent.com/22608780/101276310-00bd5500-37d2-11eb-8f65-63d5a70257b2.png" width="300"/>. <img src="https://user-images.githubusercontent.com/22608780/101276314-02871880-37d2-11eb-82b2-82fbd75b5677.png" width="300"/>

#### To test
- Create atleast 2 shipping labels for an order by following the instructions from `p91TBi-3xD`.
- Refund one of the shipping label within 2 minutes of creating the shipping label.
- Open the app and click on the order. 
  - Notice that the products card is displayed with products not associated with any shipping labels. 
  - Notice that the refunded shipping label does not display the products section.
  - Notice that the shipping label that was not refunded displays the products section.

Update release notes:
